app/controllers/public_recipes_controller.rb

```diff
@@ -0,0 +1,6 @@
+class PublicRecipesController < ApplicationController
+  def index
+    # @recipes = Recipe.where(public: true)
```

Comment:
[OPTIONAL]
Good job so far @jonathanmunamire , it will be nice if you remove comments from your code. This will make your code neat and readable.